content/kubernetes/re-clusters/multi-namespace.md

```diff
@@ -0,0 +1,176 @@
+---
+Title: Manage databases in multiple namespaces
+linktitle: Manage multiple namespaces
+description: Redis Enterprise for Kubernetes allows you to deploy to multiple namespaces within your Kubernetes cluster. This article shows you how to configure your Redis Enterprise cluster to connect to databases in multiple namespaces
+weight: 17
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/re-clusters/multi-namespace.md,
+    /kubernetes/re-databases/multi-namespace/,
+]
+
+---
+
+A single Redis Enterprise cluster (REC) can manage multiple Redis Enterprise database objects (REDB) in multiple namespaces within the same Kubernetes cluster.
+
+To learn more designing a multi-namespace Redis Enterprise cluster, see [flexible deployment options]({{<relref "/kubernetes/deployment/deployment-options.md">}}).
```

Comment:
"To learn more designing"...

By no means I'm an English language authority, but this kinda rings weird to my ears...

Maybe: "To learn more about multi-namespace..."?
Or, anything else you see fit :-)

Reply:
Good catch! I meant to write "To learn more **about** designing..."